### DISCUSSION TITLE
How I can tab multi time?

### DISCUSSION THREAD

**Discussion by vando2108** [hrsh7th/nvim-cmp#1245] [Q&A]
Hello guys, I'm using Tab to move between options, but I have a problem here. When I press <Tab> for the second time the suggestion box will appear and the second Tab will select the suggestion box.  Please let me know how I can fix this, thank you. (Sorry for my bad English)
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/35914996/196367090-5046afb9-89d9-4ec8-9438-8d7bfea3cbb9.png">

**Comment by fitrh** [comment]
Sorry, I'm a bit confused here, so what kind of behavior you want?

**Reply by vando2108** [reply]
<img width="458" alt="image" src="https://user-images.githubusercontent.com/35914996/196371180-c369b67f-abaf-4c4a-a817-40e7105202ae.png">
I press the tab for the first time, and the cursor will indent, but the second time I press the tab the suggestion box will appear like the picture, and for the next tab will move between the option in this box. But what I want is just indent multi-time without a show suggestion box.

**Reply by fitrh** [reply]
Just remove your `['<Tab>'] = ...` mapping

**Comment by yutkat** [comment]
Remove this setting.

```lua
['<Tab>'] = function(fallback)
	if cmp.visible() then
		cmp.select_next_item()
	else
		fallback()
	end
end,
```
